README.md

```diff
@@ -28,6 +28,8 @@ Django-fido provides basic components for FIDO 2 authentication - model to store
           url(r'', include('django_fido.urls')),
        ]
 
+4. Set string variable `DJANGO_FIDO_RP_NAME`.
```

Comment:
This is, strictly speaking, only necessary for `python-fido >= 0.15`. Might be worth mentioning.

Reply:
I added a note.